src/routes/console/project-[project]/storage/bucket-[bucket]/settings/+page.svelte

```diff
@@ -69,78 +69,72 @@
         } else isExtensionsDisabled = true;
     }
 
-    async function toggleBucket() {
-        try {
-            await sdkForProject.storage.updateBucket(
-                $bucket.$id,
-                $bucket.name,
-                undefined,
-                undefined,
-                enabled
-            );
-            invalidate(Dependencies.BUCKET);
-            addNotification({
-                message: `${$bucket.name} has been updated`,
-                type: 'success'
-            });
-            trackEvent('submit_bucket_enable', {
-                value: enabled
-            });
-        } catch (error) {
-            addNotification({
-                message: error.message,
-                type: 'error'
-            });
-        }
+    interface IUpdateBucket {
+        id: string;
+        name: string;
+        permissions?: string[];
+        fileSecurity?: boolean;
+        enabled?: boolean;
+        maximumFileSize?: number;
+        allowedFileExtensions?: string[];
+        compression?: string;
+        encryption?: boolean;
+        antivirus?: boolean;
```

Comment:
Is this needed considering `Models.Bucket` from `import type { Models } from '@aw-labs/appwrite-console';` exists? Maybe the type can be built using the model?